core/modules/util/ThreadPool.cc

```diff
@@ -258,4 +289,35 @@ void ThreadPool::waitForResize(int millisecs) {
     }
 }
 
+void ThreadPool::_incrPoolThreadCount() {
+    std::lock_guard<std::mutex> lockPool(_mxPool);
+    ++_poolThreadCount;
+    LOGS(_log, LOG_LVL_WARN, "&&&WARN incr _poolThreadCount=" << _poolThreadCount);
+}
+
+void ThreadPool::_decrPoolThreadCount() {
+    {
+        std::lock_guard<std::mutex> lockPool(_mxPool);
+        --_poolThreadCount;
+    }
+    LOGS(_log, LOG_LVL_WARN, "&&&WARN decr _poolThreadCount=" << _poolThreadCount);
```

Comment:
Since `_decrPoolThreadCount ` is a normal  and probably rather frequent operation then reporting it at level `WARN` is may be a bit of an "over-kill" for the production code. Perhaps `DEBUG` or even `TRACE` would still work?